client/src/components/RegistrationFormWithContinuedFlow.js

```diff
@@ -15,7 +14,19 @@ function RegistrationFormWithContinuedFlow() {
     zipCode: 0,
   });
 
-  const addFormHandler = (event) => {
+  const [backendData, setBackendData] = useState([{}]);
```

Comment:
why do you need to make this call here?

Reply:
The axios.get() returns a promise variable, made this call to consume the data from the promise variable.

Reply:
The call itself is fine, let's update the endpoint, and initiate a POST request from here to the backend, and update the route handling logic in routes.js  in backend correspondingly